ts/src/binance.ts

```diff
@@ -2598,9 +2598,9 @@ export default class binance extends Exchange {
          * @method
          * @name binance#fetchTime
          * @description fetches the current integer timestamp in milliseconds from the exchange server
-         * @see https://binance-docs.github.io/apidocs/spot/en/#check-server-time       // spot
-         * @see https://binance-docs.github.io/apidocs/futures/en/#check-server-time    // swap
-         * @see https://binance-docs.github.io/apidocs/delivery/en/#check-server-time   // future
+         * @see https://developers.binance.com/docs/binance-spot-api-docs/rest-api#check-server-time                            // spot
```

Comment:
@Dan-krm we still have hundreds of links using `binance-docs`, don't we need to update them as well?

![image](https://github.com/ccxt/ccxt/assets/43336371/825e7fe3-fe7f-4958-813f-4371275a3f92)

Reply:
I was planning on updating the rest of the links in the method docstrings in a separate PR but I can just add those changes to this PR

Reply:
I've finished updating the docstring links

Reply:
got it, thanks